repo/gen/gormGen.go

```diff
@@ -25,9 +26,11 @@ func main() {
 
 	// Generate basic type-safe DAO API for struct `model.User` following conventions
 	g.ApplyBasic(auth.UserToken{})
+	g.ApplyBasic(model.Video{})
```

Comment:
This might be redundant, updating it to `g.ApplyBasic(auth.UserToken{}, model.Video{})` would be better, please update other similar calls, I'll mark them with _redundant calls_.